gems/pending/postgres_ha_admin/failover_monitor.rb

```diff
@@ -65,8 +66,23 @@ def active_servers_conninfo
       servers.map! { |info| db_yml_params.merge(info) }
     end
 
+    def failover_successful
+      queue_failover_executed_event
+    end
+
     private
 
+    def queue_failover_executed_event
+      connection = pg_connection(@database_yml.pg_params_from_database_yml)
+      connection.exec(<<-SQL)
```

Comment:
@gtanzillo we can create some file if failover executed and before server started  and check presence of this file in  ```MiqServer#start```,  is it what you have in mind  ?